WordPress/src/main/java/org/wordpress/android/ui/compose/components/EmptyContentM3.kt

```diff
@@ -0,0 +1,132 @@
+package org.wordpress.android.ui.compose.components
+
+import android.content.res.Configuration
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.R
+import org.wordpress.android.ui.compose.theme.AppThemeM3
+import org.wordpress.android.ui.compose.unit.FontSize
+import org.wordpress.android.ui.compose.unit.Margin
+
+private fun Modifier.emptyContentTextModifier() = padding(horizontal = 30.dp)
+
+/**
+ * Reusable Material 3 component for empty screen states
+ *
+ * @param modifier [Modifier] applied on the containing [Box]
+ * @param title [String] that will be displayed as title
+ * @param subtitle (optional) [String] that will be displayed as subtitle
+ * @param image (optional) Drawable resource ID for the image that will be displayed
```

Comment:
We might rely upon more price type descriptions rather than an "optional" note. Feel free to disagree and ignore. 

```suggestion
 * @param subtitle [String?] that will be displayed as subtitle
 * @param image [Int?] Drawable resource ID for the image that will be displayed
```